pyproject.toml

```diff
@@ -15,7 +15,7 @@ authors = ["André Duarte <andre@switch-ev.com>",
 python = "^3.9"
 environs = "^9.5.0"
 cryptography = "41.0.1"
-pydantic = "^1.9.0"
+pydantic = ">=1.9,<3.0"
```

Comment:
we should update pydantic, but to a fix version and not like to an interval one like this..